index.js

```diff
@@ -0,0 +1,153 @@
+const inputSection = document.querySelector('.book-input');
+const contactSection = document.querySelector('.contact-section');
+
+// eslint-disable-next-line max-classes-per-file
+
+class Collection {
+  // eslint-disable-next-line no-unused-vars
+  constructor(books) {
+    this.books = [];
+  }
+
+    add = (book) => {
+      if (this.books === null) {
+        this.books = [];
+      }
+      this.books.push(book);
+      localStorage.setItem('Collection', JSON.stringify(this.books));
+      UI.addBookToList(book);
+    }
+
+    remove = (target) => {
+      const book = target.parentElement.parentElement;
+
+      this.books = JSON.parse(localStorage.getItem('Collection'));
+
+      this.books.forEach((item, index) => {
+        if (book.firstElementChild.innerHTML === (`"${item.title}" by ${item.author}`)) {
+          this.books = this.books.filter((i) => i !== this.books[index]);
+        }
+      });
+      localStorage.setItem('Collection', JSON.stringify(this.books));
+      UI.deleteBook(book);
+    }
+}
+
+const myCollection = new Collection();
+
+list.addEventListener('click', (e) => {
+  const h = document.querySelector('h1');
+  h.innerHTML = 'All awesome books';
+
+  list.style.color = 'blue';
+  addList.style.color = 'black';
+  contact.style.color = 'black';
+
+  dateTime.style.display = 'block';
+  displaySection.style.display = 'block';
+  inputSection.style.display = 'none';
+  contactSection.style.display = 'none';
+  e.preventDefault();
+});
+
+const deletebtn = document.querySelectorAll('.remove');
+
+deletebtn.forEach((btn) => {
+  btn.addEventListener('click', (e) => {
+    if (e.target.class === 'remove') {
+      // UI.deleteBook(e.target);
+      myCollection.remove(e.target);
+    }
+    e.preventDefault();
+  });
+});
+
+// for the add section of the page
+addList.addEventListener('click', (e) => {
+  const h = document.querySelector('h1');
+  h.innerHTML = 'Add a new book';
+
+  addList.style.color = 'blue';
+  list.style.color = 'black';
+  contact.style.color = 'black';
+  dateTime.style.display = 'block';
+  displaySection.style.display = 'none';
+  contactSection.style.display = 'none';
+  inputSection.style.display = 'block';
+  inputSection.innerHTML = `
+                          <br>
+                          <input type="text" name="title"  class ="title" placeholder="Title" required>
+                          <br><br>
+                          <input type="text" name="title"  class="author" placeholder="Author" required>
+                          <br><br>
+                          <button type="submit" class="btn">Add</button>
+`;
+  e.preventDefault();
+
+  // for add button
+  const addbtn = document.querySelector('.btn');
+  addbtn.addEventListener('click', (e) => {
+    const title = document.querySelector('.title').value;
+    const author = document.querySelector('.author').value;
+    if (title === '' || author === '') {
+      // eslint-disable-next-line no-alert
```

Comment:
Please do not disable ESLint rules unless they are in the Luxon file or are related to the number of classes per file.